rxui-sample-java/src/main/java/com/artemzin/rxui/sample/java/MainPresenter.java

```diff
@@ -41,25 +41,27 @@ Subscription bind(MainView view) {
                 .combineLatest(login, password, (l, p) -> Triplet.with(l, p, !l.isEmpty() && !p.isEmpty()))
                 .publish();
 
-        Observable<Void> signInEnable = credentials
+        Observable<Object> signInEnable = credentials
                 .filter(creds -> creds.getValue2())
-                .map(enable -> null);
+                .map(enable -> new Object());
```

Comment:
Well, I'm just following what looks to be RxBinding strategy for data-less events like clicks which were `Void` with RxJava1